homeassistant/components/statistics/sensor.py

```diff
@@ -230,42 +311,33 @@ def should_poll(self):
     @property
     def extra_state_attributes(self):
         """Return the state attributes of the sensor."""
-        if not self.is_binary:
-            return {
-                ATTR_SAMPLING_SIZE: self._sampling_size,
-                ATTR_COUNT: self.count,
-                ATTR_MEAN: self.mean,
-                ATTR_MEDIAN: self.median,
-                ATTR_QUANTILES: self.quantiles,
-                ATTR_STANDARD_DEVIATION: self.stdev,
-                ATTR_VARIANCE: self.variance,
-                ATTR_TOTAL: self.total,
-                ATTR_MIN_VALUE: self.min,
-                ATTR_MAX_VALUE: self.max,
-                ATTR_MIN_AGE: self.min_age,
-                ATTR_MAX_AGE: self.max_age,
-                ATTR_CHANGE: self.change,
-                ATTR_AVERAGE_CHANGE: self.average_change,
-                ATTR_CHANGE_RATE: self.change_rate,
-            }
+        if self.is_binary:
+            return None
+        return {
+            "configuration_"
+            + CONF_SAMPLES_MAX_BUFFER_SIZE: self._samples_max_buffer_size,
+            "configuration_" + CONF_MAX_AGE: str(self._samples_max_age),
+            "configuration_" + CONF_STATE_CHARACTERISTIC: self._state_characteristic,
```

Comment:
Without these the user will quickly lose clarity about how the sensor is setup -> what it offers.

Reply:
On second thought, I agree. This breaks with the norm and the user would normally put what is relevant into the entity `name`.